src/app/store/slices/storage/storage.selectors.ts

```diff
@@ -22,10 +21,7 @@ const storageSelectors = {
     return state.storage.namePath.reduce((t, path) => `${t}${path.name}/`, '/');
   },
   bucket(state: RootState): string {
-    const { team } = state.team;
-    const isTeam: boolean = sessionSelectors.isTeam(state);
-    // TODO: NEED TO CHANGE THIS BY WORKSPACE BUCKET
-    return (isTeam ? team?.bucket : state.user.user?.bucket) ?? '';
+    return state.user.user?.bucket ?? '';
```

Comment:
Is it not necessary to check if the workspace is selected in order to use the corresponding bucket? I mean for the TODO

Reply:
I think the comment is old thinking in a way how it would work but it is not necessary